crypto/cpuid.c

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 1998-2020 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Not sure why you went backwards with the date here :)

Reply:
I didn't...  I originally did this stuff last year, and didn't think to look now.

Reply:
I would argue that you added this file just yesterday (1998-2021)

Reply:
Fixed

Reply:
> I would argue that you added this file just yesterday (1998-2021)

Then you would be wrong.

> commit c1c52b522572327fca4972bacef37e032e0a73ed
> Author: Richard Levitte <levitte@openssl.org>
> Date:   Wed Mar 11 17:38:46 2020 +0100